site/learn/Learn-Queries.md

```diff
@@ -165,6 +165,24 @@ Variable definitions can be optional or required. In the case above, since there
 To learn more about the syntax for these variable definitions, it's useful to learn the GraphQL schema language. The schema language is explained in detail on the Schema page.
 
 
+### Default Variables
+
+Default values can also be assigned to the variables in the query by adding the default value after the type declaration. 
+
+```graphql
+query HeroNameAndFriends($episode: Episode = "JEDI") {
+  hero(episode: $episode) {
+    name
+    friends {
+      name
+    }
+  }
+}
+
+```
```

Comment:
Ok I'll fix it :] Does the rest of it look ok? I just want to make sure I'm not missing anything.